web/frontend/angular/src/app/main-pages/register/register.component.ts

```diff
@@ -4,6 +4,7 @@ import {Validators} from "@angular/forms"
 import {FormBuilder} from "@angular/forms"
 import {Router} from "@angular/router"
 import {AuthService} from "src/app/auth/auth.service"
+import * as $ from "jquery"
```

Comment:
You never installed jQuery, so this import fails.

Honestly, I'm not a fan of jQuery. I believe modern JavaScript is perfectly capable of achieving the same things in most cases (albeit with a more verbose syntax sometimes). jQuery feels like bloat to me. Is it even beneficial given that we're using Angular as a framework? Does Angular not have a better way of doing this? Is it possible for you to rewrite your error handling code without using jQuery?

Reply:
@alinaah You did not address this comment.

Reply:
Here are some alternatives to using jQuery for this. Please do further research on them, since I just gathered these from a quick search without really reading through them.

* https://www.npmjs.com/package/ngx-form-validations
* https://www.pluralsight.com/guides/how-to-display-validation-messages-using-angular
* https://stackoverflow.com/questions/52771445/best-way-to-show-error-messages-for-angular-reactive-forms-one-formcontrol-mult
* https://netbasal.com/make-your-angular-forms-error-messages-magically-appear-1e32350b7fa5

Reply:
Since we will have many forms elsewhere, it'd be good to make this error handling re-usable in some way across all forms.